caffe2/CMakeLists.txt

```diff
@@ -1033,6 +1033,10 @@ if(TRACING_BASED AND NOT BUILD_LITE_INTERPRETER AND NOT INTERN_BUILD_MOBILE)
   )
 endif()
 
+message(STATUS "Tracing Based Flag: ${TRACING_BASED}")
+message(STATUS "Build Lite Interpreter Flag: ${BUILD_LITE_INTERPRETER}")
+message(STATUS "Intern Build Mobile Flag: ${INTERN_BUILD_MOBILE}")
```

Comment:
Delete debugging code

Reply:
I saw a bunch of places in existing that print the relevant flags where they are used as `STATUS`, so I wanted to keep there here - do you think that's okay?